backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/ddl/public/functions/after_migration_sync_translations.sql

```diff
@@ -0,0 +1,23 @@
+
+DROP FUNCTION IF EXISTS after_migration_sync_translations();
+
+
+CREATE FUNCTION after_migration_sync_translations() RETURNS void
+    LANGUAGE plpgsql
+AS
+$$
+DECLARE
+    count_updated NUMERIC;
+BEGIN
+    PERFORM update_TRL_Tables_On_AD_Element_TRL_Update();
+
+    RAISE NOTICE 'Synchronized translations';
+
+END;
+$$
+;
+
+ALTER FUNCTION after_migration_sync_translations() OWNER TO metasfresh
```

Comment:
no need to alter owner. It's assumed we run with metasfresh. so u can remove this line.